### PR TITLE
Automatically use latest RHCOS build

### PR DESCRIPTION
**Pull request by vrutkovs**
Rework #192 w/ additional fixes

Not sure if permissive selinux is required though

### CODE REVIEW

**Comment by derekhiggins** [comment]
Build FAILURE, see build http://10.8.144.11:8080/job/dev-tools/471/

**Comment by derekhiggins** [comment]
Build FAILURE, see build http://10.8.144.11:8080/job/dev-tools/477/

**Comment by derekhiggins** [comment]
Build FAILURE, see build http://10.8.144.11:8080/job/dev-tools/478/

File: common.sh [hunk 1]
```diff
@@ -39,14 +39,20 @@ NODES_PLATFORM=${NODES_PLATFORM:-"libvirt"}
...
+export RHCOS_IMAGE_VERSION="${RHCOS_IMAGE_VERSION:-${RHCOS_LATEST}}"
+export RHCOS_IMAGE_FILENAME_OPENSTACK_GZ="$(curl ${RHCOS_IMAGE_URL}/${RHCOS_IMAGE_VERSION}/meta.json | jq -r '.images.openstack.path')"
+export RHCOS_IMAGE_NAME=$(echo $RHCOS_IMAGE_FILENAME_OPENSTACK_GZ | sed -e 's/-openstack.*//')
+# FIXME(shardy) note the -openstack image doesn't work for libvirt
+# as the qemu ignition config injection described in the docs at
+# https://coreos.com/os/docs/latest/booting-with-libvirt.html
+# doesn't work - probably we need to download both as the
+# -openstack one may be needed for the baremetal nodes so we get
+# config drive support, or perhaps a completely new image?
+export RHCOS_IMAGE_FILENAME="${RHCOS_IMAGE_NAME}-qemu.qcow2"
```
Comment (hardys) [inline common.sh]: The -qemu image is no longer needed ref https://github.com/openshift-metalkube/dev-scripts/pull/226/commits/2967acaa701c0dfc5fe44deb656a7e793418ecbc

File: assets/files/etc/selinux/config [hunk 2]
```diff
@@ -0,0 +1,11 @@
+# This file controls the state of SELinux on the system.
+# SELINUX= can take one of these three values:
+#     enforcing - SELinux security policy is enforced.
+#     permissive - SELinux prints warnings instead of enforcing.
+#     disabled - No SELinux policy is loaded.
+SELINUX=permissive
```
Comment (hardys) [inline assets/files/etc/selinux/config]: Do we have a bug reference to track fixing the policy issues that make this necessary?

Probably we should have a FIXME here to reference it?

**Comment by hardys** [comment]
This is now working for me locally - there's one small rebase mistake which we should probably fix before merging, and it'd be nice to get a green CI run :)

File: common.sh (hunk 1 above)
Comment (vrutkovs) [reply common.sh]: Removed that, thanks

**Comment by derekhiggins** [comment]
Build SUCCESS, see build http://10.8.144.11:8080/job/dev-tools/486/

**Comment by hardys** [comment]
Yay, passing CI :)

**Pull request merged**